cli/src/cmd/root.go

```diff
@@ -29,5 +30,9 @@ func init() {
 	// TODO: read the docs for cobra.OnInitialize() and decide if it's needed
 	cobra.OnInitialize()
 
+	rootCmd.PersistentFlags().StringVar(&state.ConfigFile, "config", "", "config file (default is $WORKING_DIR/config.yaml)")
```

Comment:
I think the default is config.yml and not config.yaml

Reply:
It's actually both, the program will accept config.yaml if it exists, and config.yml if that exists.